pkg/apply/taskrunner/context.go

```diff
@@ -13,14 +13,15 @@ import (
 )
 
 // NewTaskContext returns a new TaskContext
-func NewTaskContext(eventChannel chan event.Event, resourceCache cache.ResourceCache) *TaskContext {
+func NewTaskContext(eventChannel chan event.Event, resourceCache cache.ResourceCache, continueOnError bool) *TaskContext {
```

Comment:
Why do we need the continue one error bool in the TaskContext?
Isn't it only needed by the baseruneer?

Reply:
> Why do we need the continue one error bool in the TaskContext?
> Isn't it only needed by the baseruneer?

This is from an older version. Please review the latest version.